lib/livingdocs_connector.js

```diff
@@ -0,0 +1,68 @@
+const _ = require('lodash')
+const axios = require('axios')
+const {mapMetadata} = require('../mappers/image_mapper')
+const documentMapper = require('../mappers/document_mapper')
+const mapToAbstractLivingdoc = require('../mappers/map_dpa_article_to_dpa_design')
+
+module.exports = {
```

Comment:
It would be nice to have that in the sdk.

Reply:
Currently, the SDK does not support POST request. I'd need some input from core devs first to change this.